Tasks/AppCenterDistributeV3/appcenterdistribute.ts

```diff
@@ -296,11 +296,11 @@ function getBranchName(ref: string): string {
  * If the input is a single folder, zip it's content. The archive name is the folder's name
  * If the input is a set of folders or files, zip them so they appear on the root of the archive. The archive name is the parent folder's name.
  */
-function prepareSymbols(symbolsPaths: string[]): Q.Promise<string> {
+function prepareSymbols(symbolsPaths: string[], forceArchive: boolean): Q.Promise<string> {
```

Comment:
I think it might be better to initialize `forceArchive` with a default parameter here.